lib/task/import/validate/csvBaseValidator.class.php

```diff
@@ -91,9 +94,45 @@ public function appendDuplicatedColumnError(string $columnName)
         $this->testData->addResult(sprintf('Unable to validate because of duplicated columns in CSV.'));
     }
 
+    public function setHeader($header)
+    {
+        $this->header = $header;
+    }
+
+    // Specify required columns to trigger basic checks - presence, duplication.
+    public function setRequiredColumns(array $requiredColumns)
+    {
+        $this->requiredColumns = $requiredColumns;
+    }
+
     public function testRow(array $header, array $row)
     {
         ++$this->rowNumber;
+
+        // Save copy of header when processing first row of CSV.
+        if (0 == count($this->header)) {
```

Comment:
Opinion: I think it's better to test `null === $this->header` or `empty($this->header)` instead using count.

Reply:
👍